manager/integration/tests/common.py

```diff
@@ -4988,7 +4988,8 @@ def check_recurring_jobs(client, recurring_jobs):
         assert recurring_job.cron == spec["cron"]
 
         expect_retain = spec["retain"]
-        if recurring_job.task == "snapshot-cleanup":
+        if recurring_job.task == "snapshot-cleanup" or \
+                recurring_job.task == "filesystem-trim":
```

Comment:
nit: If having more than one check, probably can use `in` to check the task type.